packages/repository/src/index.ts

```diff
@@ -3,21 +3,21 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export * from './common-types';
-export * from './decorators/model';
-export * from './decorators/repository';
-export * from './decorators/relation';
+export * from './connectors/connector';
+export * from './connectors/crud.connector';
+export * from './connectors/kv.connector';
+export * from './decorators/model.decorator';
+export * from './decorators/repository.decorator';
+export * from './decorators/relation.decorator';
 export * from './decorators/metadata';
+export * from './mixins/mixin';
+export * from './mixins/repository.mixin';
+export * from './repositories/kv.repository';
+export * from './repositories/legacy-juggler-bridge';
```

Comment:
You can add a `index.ts` in the `/repositories` folder and export these from there, and then just do a `export * from './repositories'` here. It is the pattern we advocate in user applications.

Reply:
Should I do the same for every other artifact folders in the monorepo?

Reply:
Yea sure :)